src/Integration/LocalServices/ProjectSystemHelper.cs

```diff
@@ -45,6 +44,7 @@ internal class ProjectSystemHelper : IProjectSystemHelper
         internal const string VbCoreProjectKind = "{778DAE3C-4631-46EA-AA77-85C1314464D9}";
         internal const string CSharpCoreProjectKind = "{9A19103F-16F7-4668-BE54-9A1E7A4F7556}";
         internal const string CppProjectKind = "{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}";
+        internal const string NodeJSProjectKind = "{9092aa53-fb77-4645-b42d-1ccca6bd08bd}";
```

Comment:
This new constant is currently only used in the tests. However, I'll leave it here for consistency.